demux/utils/novaseq_samplesheet.py

```diff
@@ -185,7 +206,7 @@ def pad_and_rc_indexes(
 
         return index1, index2
 
-    def get_reagent_kit_version(self) -> int:
+    def get_reagent_kit_version(self) -> Optional[float]:
```

Comment:
Consider to raise exception if reagent kit version is not known, or add those other values

Reply:
fixed in https://github.com/Clinical-Genomics/demultiplexing/pull/146/commits/aba742c51a4ac99eb9d4f81cf0cc141c27d6eaa0